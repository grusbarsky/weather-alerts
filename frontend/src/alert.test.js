import React from "react";
import { render } from "@testing-library/react";
import Alert from "./alert";

it("renders without crashing", function() {
  render(<Alert />);
});

it("matches snapshot for danger", function() {
  let messages = ["DANGER"];
  const { asFragment } = render(<Alert type="danger" messages={messages} />);
  expect(asFragment()).toMatchSnapshot();
});

it("matches snapshot for success", function() {
  let messages = ["SUCCESS"];
  const { asFragment } = render(<Alert type="success" messages={messages} />);
  expect(asFragment()).toMatchSnapshot();
});