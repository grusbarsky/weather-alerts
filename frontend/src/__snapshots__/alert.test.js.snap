// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot for danger 1`] = `
<DocumentFragment>
  <div
    class="alert-danger"
    role="alert"
  >
    <p
      class="alert-text text-center"
    >
      DANGER
    </p>
  </div>
</DocumentFragment>
`;

exports[`matches snapshot for success 1`] = `
<DocumentFragment>
  <div
    class="alert-success"
    role="alert"
  >
    <p
      class="alert-text text-center"
    >
      SUCCESS
    </p>
  </div>
</DocumentFragment>
`;
